js/controllers.js

```diff
@@ -202,21 +188,25 @@ appControllers.controller('HomeController', ['$scope', '$http', '$location', fun
   }
   if(data["Mentee"]) {
     $scope.user.type.push("Mentee");
+    $scope.profile_title = "Your Mentor";
 
-    // $.ajax({
-    //   url: "api/mentor",
-    //   dataType: "json",
-    //   async: true,
-    //   success: function(result) {
-    //     //data = result;
-    //     $scope.myMentor = result;
-    //     console.log("getMentor");
-    //     console.log($scope.myMentor);
-    //   },
-    //   type: 'GET',
-    //   error: $scope.ajaxError
-    //   // error: ajaxError
-    // });
+    function getMentorData(mentorUsername) {
+      console.log(mentorUsername)
+      $.ajax({
+        url: "api/mentor/" + mentorUsername,
+        dataType: "json",
+        async: true,
+        success: function(result) {
+          $scope.show_identifier = true;
+          $scope.myMentor = result[0];
+          console.log("getMentor");
+          $scope.$apply();
+        },
+        type: 'GET',
+        error: $scope.ajaxError
+        // error: ajaxError
```

Comment:
Deleted that line in new commit. In a later pull request we may be able to go in and clean up similar comments in other areas.